x-pack/packages/kbn-entities-schema/index.ts

```diff
@@ -8,3 +8,6 @@
 export * from './src/schema/entity_definition';
 export * from './src/schema/entity';
 export * from './src/schema/common';
+export * from './src/rest_spec/delete';
```

Comment:
why do we set those in kbn-entities-schema instead of entity manager ?

Reply:
I feel like it's easier to reference types from a package than a plugin, especially once you start doing any type of frontend/backend work.